style.css

```diff
@@ -0,0 +1,68 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+section{
+    min-height: 100vh;
+    background: #000;
+
```

Comment:
please remove the space in css file there are lots of space are there in your css file...

Reply:
okk